OCS/deploy-ocs.sh

```diff
@@ -2,6 +2,18 @@
 NAMESPACE="openshift-storage"
 LOCALNAMESPACE="local-storage"
 
+if [ -z "${KUBECONFIG}" ]; then
+  export KUBECONFIG=$(find ${PWD%/*} -iname kubeconfig -type f)
```

Comment:
nice, maybe it s cleaner to go with $HOME or . ?

Reply:
Thanks, updated to ${HOME}